rpcserver.go

```diff
@@ -6901,6 +6901,16 @@ func (r *rpcServer) DebugLevel(ctx context.Context,
 	err := build.ParseAndSetDebugLevels(req.LevelSpec, r.cfg.LogWriter)
 	if err != nil {
 		return nil, err
+	} else {
```

Comment:
nit: You can omit the else block and write the statement within it after the if block, since we return early if there's an error.

Reply:
Done ... thanks!